src/topology/vector_bundle.lean

```diff
@@ -423,20 +521,58 @@ namespace topological_vector_bundle
 
 namespace trivialization
 
+/-- Forward map of `continuous_linear_equiv_at` (only propositionally equal),
+  defined everywhere (`0` outside domain). -/
+@[simps apply {fully_applied := ff}]
+def continuous_linear_map_at (e : trivialization R F E) (b : B) :
+  E b →L[R] F :=
+{ cont := by { dsimp, rw [e.coe_linear_map_at b],
+  refine continuous_if_const _ (λ hb, _) (λ _, continuous_zero),
+  refine continuous_snd.comp (e.to_local_homeomorph.continuous_on.comp_continuous
+    (total_space_mk_inducing R F E b).continuous (λ x, e.mem_source.mpr hb)) },
```

Comment:
IMHO, this deserves a `begin ... end` block.

Also, can you explain why you mention `(only propositionally equal)` in the docstring? At least, I hope that the `to_fun` fields are defeq?

Reply:
No, they are not defeq because the one that is defined everywhere features an `if then else`. It is defined as `if hb : b ∈ e.base_set then e.linear_equiv_at b hb else 0` so it's not defeq to `e.linear_equiv_at b hb`, it is only propeq to it when `b ∈ e.base_set.